google-map.html

```diff
@@ -371,7 +371,7 @@
       if (this.map) {
         return; // already initialized
       }
-      if (!(window.google && window.google.maps)) {
+      if (this.$.api.libraryLoaded !== true) {
```

Comment:
Agreed with Eric about the direct testing (see my comment here for more on why it will work: https://github.com/GoogleWebComponents/google-map/issues/185#issuecomment-132002611) 

v3 is almost certainly never going to move to loading basic functionality like markers asynchronously without making it transparent to consuming code anyways since basically all existing source code assumes that the API is loaded at the same time.